workers/unity/Assets/Fps/Scripts/SetupLogic/ClientWorkerConnector.cs

```diff
@@ -32,8 +32,8 @@ protected override void HandleWorkerConnectionEstablished()
         {
             var world = Worker.World;
 
-            // Only take the Heartbeat from the PlayerLifecycleConfig Client Systems.
-            world.GetOrCreateManager<HandlePlayerHeartbeatRequestSystem>();
+            PlayerLifecycleHelper.AddClientSystems(world, false);
+            PlayerLifecycleConfig.MaxPlayerCreationRetries = 0;
```

Comment:
Why do we disable retries here? Is it related to the connection screen flow?

Reply:
yup, the current flow is that user input = a retry - so we want to disable manual retrying